platform-sdk/docs/proposals/TSS-Ledger-Id/TSS-Ledger-Id.md

```diff
@@ -0,0 +1,1001 @@
+| TSS-008        | The ledger signature SHOULD be verifiable by an EVM smart contract without excessive gas cost.                                        |
+| TSS-009        | The TSS implementation SHOULD be able to switch elliptic curves.                                                                      |
+| TSS-010        | The TSS algorithm SHOULD be able to model consensus weight with high precision.                                                       |
+| TSS-011        | Minimize the number of steps a Node Operator needs to perform.                                                                        |
+
+Block Signing Requirements
+
+| Requirement ID |                                   Requirement Description                                   |
+|----------------|---------------------------------------------------------------------------------------------|
+| TSSBS-001      | The initial block in the block stream SHOULD be signable and verifiable with the ledger id. |
+| TSSBS-002      | On a new network, the initial block MUST be block 1                                         |
+
+### Design Decisions
+
+#### New Address Book Life-Cycle
+
+The TSS-Ledger-ID proposal modifies the address book and roster lifecycle presented in TSS-Roster as follows:
+
+1. The `Roster` format does not change.
+2. The app must set the `candidate roster` early enough to allow the nodes to generate the key material for the new
+   roster before the software upgrade.
+3. If the `candidate roster` does not have enough key material after a software upgrade, it will not replace the
+   `active roster` and remain the candidate roster. The platform will continue to attempt to key the candidate
+   roster after software upgrade.
+4. The app must detect whether or not the `candidate roster` became the `active roster`.
+
+![TSS Roster Lifecycle](./TSS-Roster-Lifecycle.drawio.svg)
+
+#### TSS Algorithm
+
+The threshold signature scheme chosen for our implementation is detailed in [Non-interactive distributed key
+generation and key resharing (Groth21)](https://eprint.iacr.org/2021/339). This scheme is able to meet the
+core requirements listed above. The rest of this proposal details the design of how to implement this scheme in the
+consensus nodes. See the TSS-Library proposal for more details on the Groth21.
+
+The Groth21 algorithm uses [Shamir Secret Sharing](https://en.wikipedia.org/wiki/Shamir%27s_secret_sharing) to hide the
+private key of the ledger and distribute shares of it so that a threshold number of node signatures on a message can
+generate the ledger signature on the same message. The threshold should be high enough to ensure at least 1 honest
+share is required to participate in the signing since a threshold number of shares can also collude to recover the
+ledger private key, if they so choose.
```

Comment:
I am rejecting the language change because this is a security related behavior.  We need the threshold high enough that we require honest node participation to prevent malicious nodes from being able to collude.